common/web/input-processor/build.sh

```diff
@@ -34,6 +34,7 @@ builder_describe "Builds the standalone, headless form of Keyman Engine for Web'
   "--ci        Sets ${BUILDER_TERM_START}test${BUILDER_TERM_END} action to use CI-based test configurations & reporting"
 
 builder_describe_outputs \
+  configure          /node_modules \
```

Comment:
So, a fun tidbit about this tweak:

When `builder` dependency management checks against this parameterization, if there is not a general `configure` setting here, it will not check against the settings for specific targets (here, `configure:module` and `configure:tools`) and will instead launch the main `configure` action.  This can have a very interesting side-effect later in the build.

With the added line above temporarily removed, the related part of the build (from the top-level Web build script) looks like this, should a top-level `configure` have been run in a prior call:

![image](https://user-images.githubusercontent.com/25213402/196363230-f16636e5-4b9a-4869-92ca-3f4481e9dd6e.png)

Note the `[common/web/input-processor] configure starting` part - this triggers an npm dependency refresh... but only for _some_ of the lower-level build products.  Since the `build` actions are specific to each individual action, those don't get re-run.  So, web is compiled against lower-level build products on _older_ versions of the files than the top-level project references during its build, all while the incremental build system feels it can still reuse prior work.  This, in turn, somehow causes source-mapping to feel a need to differentiate between two versions of certain source files... and Google Closure (for minification) ends up falling over on that as a result:

![image](https://user-images.githubusercontent.com/25213402/196364429-dd1ca226-6f46-402f-b9b2-0cb3556726d4.png)

The general `configure` (with no matching target) allows the builder subsystem to bypass the npm refresh when `node_modules` already exists, which then keeps the file versions consistent for all components of the build.

------

For the Closure error above, it's due to this aspect of `tsc`'s output sourcemap:

```jsonc
"sources":[
/* ... ,*/ 
"file:///keyman/web/source/kmwtypedefs.ts",
"file:///keyman/web/source/kmwexthtml.ts",
"file:///keyman/web/source/singleton.ts",
/* ... ,*/ 
"web/source/kmwtypedefs.ts",
"web/source/kmwexthtml.ts",
"web/source/singleton.ts",
```

(excerpted from)

![image](https://user-images.githubusercontent.com/25213402/196365397-1b0ce12b-d71d-45fa-8d07-5b895a8fa98a.png)


The main Web source files are, for whatever reason, referenced twice over when this part of the builder-script configuration is malformed (missing the line added above).  The `file:///` versions that result should not exist, and that's a `tsc` byproduct that results if `node_modules` gets altered mid-build due to the improper config.

Reply:
So, it sounds like we may need to tweak this slightly in builder.inc.sh? The proposed workaround -- adding the non-specific `configure   <file>` is a good way to move us forward, but really builder should be smart enough to handle it.